packages/networks/teritori-testnet/index.ts

```diff
@@ -62,7 +62,7 @@ export const teritoriTestnetNetwork: CosmosNetworkInfo = {
     "https://explorer.teritori.com/teritori-testnet/account/$address",
   idPrefix: "testori",
   testnet: true,
-  backendEndpoint: "https://dapp-backend.testnet.teritori.com",
+  backendEndpoint: "http://192.168.1.78:9090",
```

Comment:
revert this

Reply:
My apologies for that..
https://github.com/TERITORI/teritori-dapp/pull/1339/commits/439ae84ff595053637d634cd168109bac94ff9be